src/tutorials.md

```diff
@@ -2,7 +2,7 @@
 
 % https://github.com/common-workflow-language/user_guide/issues/160
 
-This is a list to tutorials provided by the CWL community. Send a
+This is a list to tutorials provided by the CWL community. Open a
 [pull request](https://github.com/common-workflow-language/user_guide/)
 if you would like to add another tutorial to the list.
```

Comment:
Thanks for bring my attention to this text.

```suggestion
This is a list of tutorials provided by the CWL community. Use the `Edit this page` link in the menu if you would like to add another tutorial to the list.
```

This is more straightforward, than sending readers to the GitHub hosted repository without showing them which file to edit.

FYI: Some communities don't use social coding services like GitHub, so pull requests (or even the patches themselves) are **sent** via email.